.github/workflows/changelog-check.yml

```
@@ -0,0 +1,28 @@
+name: Changelog Check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  check-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        
+        # Add a comment on the PR if there's no changelog modification
+      - name: Changelog reminder
+        uses: peterjgrainger/action-changelog-reminder@v1.2.0
+        with:
+          changelog_regex: 'CHANGELOG.md'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          
+      # Gives an error if there's no change in the changelog (except using label)
+      - name: Changelog check
+        uses: dangoslen/changelog-enforcer@v1.4.0
+        with:
+          changeLogPath: 'CHANGELOG.md'
+          skipLabel: 'Skip-Changelog'
```

I created the label with a different name now (not your fault since I created it after you made the PR, but I think the "lowercase with spaces" asthetic fits the format of the other labels more):

```suggestion
          skipLabel: 'no changelog entry needed'
```

---

Ok